src/main/resources/test_files/runtime/FunctionUserDefinedDynamic/NamedFunctionRef-Error2.jq

```diff
@@ -0,0 +1,4 @@
+(:JIQS: ShouldCrash; ErrorCode="XPDY0130"; :)
+max#1
+
+(: referencing built-in functions - not supported :)
```

Comment:
Do we have an issue to keep track of this?

Reply:
I will create the issue for this. I would like to propose keeping it as an active test. If disabled, it might be forgotten/unnoticed at the time of working on this issue. In the current state, it will cause an error which will raise the need for updating it once the correct implementation is in place. Does that make sense?

Reply:
Yes -- please just make sure we remember to fix this and implement references to builtin functions later on. Thanks!

Reply:
Of course. I just want to point out here that the reason why referencing isn't supported on built-in types is directly based on the fact that they don't fit into to functionItem format that clearly defines body, arguments, signature and so on. Once built-in functions are refactored to a structure that works with these concepts, it should simply be a matter of plugging them in.

Reply:
Issue created: https://github.com/RumbleDB/rumble/issues/346